autopts/ptsprojects/boards/__init__.py

```diff
@@ -204,8 +211,8 @@ def get_tty(debugger_snr):
     for dev in devices.keys():
         if devices[dev] == debugger_snr:
             tty = dev
-            break
```

Comment:
why keep iterating here?

Reply:
This may not be needed, but at the time I was adding that change I seem to remember that there was problems where it was picking the UART for the wrong core of the nRF5340 chip.

Reply:
I added the conditional check, with a comment which should spread some light on why the change is needed to support the nRF5340 Audio devkit.